deploy-config/front-end-nginx-config/nginx.conf

```diff
@@ -1,7 +1,7 @@
 worker_processes 1;
 daemon off;
 
-error_log stderr;
+error_log stderr info;
```

Comment:
Is there a way we can raise the level of the log message being produced in the csp rather than lower the level that gets logged to stderr?